examples/viewer_fps_label.py

```diff
@@ -23,5 +23,8 @@ def update_fps(fps):
 # without warning in future versions!
 viewer.window._qt_viewer.canvas._scene_canvas.measure_fps(callback=update_fps)
 
+# call update_fps function once to have it show before
+# mouse interaction with the canvas
+update_fps(60)
```

Comment:
Why stop at `60`? napari is supposed to be FAST 🚀 !
Let's set this to atleast `144` 😜